### PR TITLE
Fixed 'headcount per FTE' calculations in `CensusResponse` builder

### PR DESCRIPTION
**Pull request by WolfyUK**
### Context
[AB#215339](https://dfe-ssp.visualstudio.com/a14e55df-4fbf-4a2f-a11d-22b187178343/_workitems/edit/215339) [AB#215051](https://dfe-ssp.visualstudio.com/a14e55df-4fbf-4a2f-a11d-22b187178343/_workitems/edit/215051)

### Change proposed in this pull request
Modified view to include 'headcount' equivalent of each 'FTE' value for use in response builder calculations. Test coverage over calculations.

### Guidance to review 
Side-by-side comparison of dev and local should show identical values for all Census dimensions apart from 'headcount per FTE', which should now be resolved. e.g.:

Screenshot_2-7-2024_9190_s198d01-education-benchmarking-d8a2cna4f9btahaz a02 azurefd net

image

### Checklist
- [x] Work items have been linked [(use AB#)](https://learn.microsoft.com/en-us/azure/devops/boards/github/link-to-from-github?view=azure-devops#use-ab-to-link-from-github-to-azure-boards-work-items)
- [x] Your code builds clean without any errors or warnings
- [x] You have run all unit/integration tests and they pass
- [x] Your branch has been rebased onto main
- [x] You have tested by running locally

### CODE REVIEW

**Review by jrabbott — CHANGES_REQUESTED** [review]
History view will need altering too

File: platform/src/apis/Platform.Api.Insight/Census/Census.cs
Comment (jrabbott) [inline platform/src/apis/Platform.Api.Insight/Census/Census.cs:1]: These new categories appear to be unused.

File: platform/src/apis/Platform.Api.Insight/Census/CensusResponse.cs
Comment (jrabbott) [inline platform/src/apis/Platform.Api.Insight/Census/CensusResponse.cs:1]: Given this change, I think the response props need to drop "FTE" from the end.

**Review by jrabbott — COMMENTED** [review]
History view will need altering too

File: platform/src/apis/Platform.Api.Insight/Census/Census.cs
Comment (WolfyUK) [reply platform/src/apis/Platform.Api.Insight/Census/Census.cs:1]: Reverted

**Review by jrabbott — APPROVED** [review]

**Pull request merged**